app/javascript/dashboard/api/inbox/conversation.js

```diff
@@ -112,6 +112,12 @@ class ConversationApi extends ApiClient {
     return axios.post(`${this.url}/${conversationId}/transcript`, { email });
   }
 
+  updateDraftMessage({ conversationId, message }) {
```

Comment:
As mentioned above, let us move draftMessages from the conversation namespace to its own namespace.